src/components/header/Header.js

```diff
@@ -25,7 +25,7 @@ function Header() {
   return (
     <Headroom>
       <header className={isDark ? "dark-menu header" : "header"}>
-        <a href="" className="logo">
+        <a href="/" className="logo">
```

Comment:
this should be better.
```suggestion
        <a href="#" className="logo">
```

Reply:
I actually was going to go with "#" but I think in this case "/"  would be better as it will refresh the page, which is the point of clicking on a "home" button. 

Let me know what you think about this.

Reply:
Ok, this looks good then. :)